templates/native/react-redux/test/template.reducer.spec.js

```diff
@@ -76,4 +76,34 @@ describe('TEMPLATE_LOWER_CAMEL_CASE_NAMEReducer - Unit Test', () => {
     each of the asynchronous action's scenarios in its own 'it'. Remember to test happy and sad cases.
     If you have no asynchronous actions, this code may be omitted.
     */
+    describe('exampleAsyncAction', () => {
+        it('should dispatch exampleAction', () => {
+            /*
+             Mock dispatch and getState
+             Note: jest 23 - you may use .mockName this helps to indicate which mock function is being referenced.
+
+             const dispatch = jest.fn().mockName('getState');;
+
+             const dispatch = jest.fn().mockName('dispatch');;
+
+             */
+            const getState = jest.fn(() => ({
```

Comment:
I still like the `mockImplimentation` chain. I think it makes it more clear to users.